### PR TITLE
ENH better packages

### PR DESCRIPTION
**Pull request by beckermr**

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -69,7 +69,7 @@
                         <a class="page-scroll" href="https://conda-forge.org/#contribute">Contribute</a>
                     </li>
                     <li>
-                        <a href="https://conda-forge.org/feedstocks">Packages</a>
+                        <a href="https://conda-forge.org/https://conda-forge.org/feedstock-outputs">Packages</a>
```
Comment (pkgw) [inline 404.html:72]: Is this correct?

Comment (beckermr) [reply 404.html:72]: yup.

Comment (beckermr) [reply 404.html:72]: oh woops

Comment (beckermr) [reply 404.html:72]: Good catch!

**Pull request merged**